server.py

```diff
@@ -382,8 +382,12 @@ def system_info():
 
 @route('/splash_page')
 def splash_page():
-    my_ip = get_node_ip()
-    if my_ip:
+    try:
+        my_ip = get_node_ip()
+    except Exception as e:
+        ip_lookup = False
+        url = e
```

Comment:
`url` seems to be an odd choice for this variable. How about `error_msg` or something like that instead?